test/parallel/test-assert.js

```diff
@@ -144,10 +144,13 @@ if (typeof Symbol === 'symbol') {
 }
 
 // primitive wrappers and object
-assert.doesNotThrow(makeBlock(a.deepEqual, new String('a'), ['a']), a.AssertionError);
-assert.doesNotThrow(makeBlock(a.deepEqual, new String('a'), {0: 'a'}), a.AssertionError);
-assert.doesNotThrow(makeBlock(a.deepEqual, new Number(1), {}), a.AssertionError);
-assert.doesNotThrow(makeBlock(a.deepEqual, new Boolean(true), {}), a.AssertionError);
+assert.throws(makeBlock(a.deepEqual, new String('a'), ['a']), a.AssertionError);
+assert.throws(makeBlock(a.deepEqual, new String('a'), {0: 'a'}), a.AssertionError);
+assert.throws(makeBlock(a.deepEqual, new Number(1), {}), a.AssertionError);
+assert.throws(makeBlock(a.deepEqual, new Boolean(true), {}), a.AssertionError);
+
+// prototypes
+assert.throws(makeBlock(assert.deepEqual, Object.create({}), {}), a.AssertionError);
```

Comment:
Sorry about that. I actually DO have a guideline, but in this case I saw some long lines just above and ignored it. Anyway, wrapped this and other offending lines.